tools/build.py

```diff
@@ -53,6 +53,9 @@
     parser.add_argument("--source", dest="source_dir", type=argparse_filestring_type,
                         default=None, help="The source (input) directory", action="append")
 
+    parser.add_argument("--custom-targets", dest="custom_targets_directory", type=argparse_filestring_type,
+                        default=None, help="Specify directory containing custom_targets.json", action="append")
```

Comment:
This should not be needed as you use `get_default_options_parser()`.

Reply:
Will fix. This would not be the case for project.py, and device_managment.py?

Reply:
both `project.py` and `device_management.py` don't use `get_default_options_parser()`. I will accept a patch that changes that, if you're interested, and I won't fault you for keeping the status quo.

Reply:
I see how this works now. 
Patch in separate MR? Any traps you can think of?

Trust me, you don't want me doing much more than copy paste and adding an ```elif``` to python sources.

Reply:
Patch in separate PR is appreciated. I can't think of any traps, and I don't want to block this PR reviewing that addition as well.